tests/integration/docs/test_docs_command.py

```diff
@@ -0,0 +1,16 @@
+from samcli.commands.docs.docs_context import DocsContext
+from tests.integration.docs.docs_integ_base import DocsIntegBase
+from tests.testing_utils import run_command
+
+
+class TestDocsCommand(DocsIntegBase):
+    def test_docs_command(self):
+        command_list = self.get_docs_command_list()
+        command_result = run_command(command_list)
+        stdout = command_result.stdout.decode("utf-8").strip()
+        expected_message = (
```

Comment:
how do we check this in a CI context? one can't open a web browser there.

Reply:
In a CI environment, the `web_browser: webbrowser.BaseBrowser = webbrowser.get(self.browser_name)` line will raise an exception that there is no browser on the machine to be found. We can't really test the browser functionality further beyond that, but we assert that we at least print the URL for the user to use manually.